react.js

```diff
@@ -5,61 +5,84 @@
 
 // 1. Write a React component that prints "I am a component!" Be sure to include all necessary imports, exports, etc.
 
+import React, { Component } from 'react'
+import './App.css';
 
+class Selfaware extends Component{
 
+  render(){
+    return(
+      <div>
+          <p>
+            I am a component!
+          </p>
+      </div>
+    )
+  }
+}
+
+export default Selfaware
 
 
 // 2. Refactor this vanilla javascript loop to a map function. The output should remain the same.
 
 var names = ["Ford Prefect", "Arthur Dent", "Trillian", "Zaphod", "Vogon", "Marvin, the Paranoid Android"]
 
-for(let i=0; i<names.length; i++){
-  console.log(`${names[i]} is ${names[i].length} characters long.`)
-}
+// for(let i=0; i<names.length; i++){
+//   console.log(`${names[i]} is ${names[i].length} characters long.`)
+// }
 
+names.map((v,i) => {
+  console.log(`${names[i]} is ${names[i].length} characters long.`);
+})
 
 
 // 3. Destructure the following variables out of state.
 
+
 this.state = {
   name: "Dracula",
   home: "Transylvania",
   dislikes: ["mirrors", "garlic", "wooden stakes"]
 }
 
-
+const { name, home, dislikes } = this.state
 
 // 4. Write a React method that would add one and update the state of the count each time the method is called.
 
+
 this.state = {
   count: 0
 }
 
-
+this.setState({count: this.state.count++})
```

Comment:
Functionality looks great. It just needs a method that is called to update the state.